drivers/robot_control/myagv_driver.py

```diff
@@ -73,35 +73,39 @@ def _map_vel(self,vel:float):
         return int(127*vel)+128
     
     def _run_thread(self):
-        x_velocity,y_velocity,rz_velocity = 0.0,0.0,0.0
```

Comment:
these changes are actually not part of what this PR is trying to achieve but I forgot to separate them out. Here we are just improving how the driver sets the speed.